From 1eeacf8cfed912eb1483e60ac7d15eea78ee7290 Mon Sep 17 00:00:00 2001
From: Catalin Udma <catalin-dan.udma@nxp.com>
Date: Tue, 13 Aug 2019 14:46:37 +0300
Subject: [PATCH] kni_net: add required include file for signal_pending

This commit fix the following error:
kern/kni_net.c:601:6: error: implicit declaration of function
'signal_pending

Signed-off-by: Catalin Udma <catalin-dan.udma@nxp.com>
---
 kern/kni_net.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kern/kni_net.c b/kern/kni_net.c
index b4cbfa9..f80f919 100644
--- a/kern/kni_net.c
+++ b/kern/kni_net.c
@@ -35,6 +35,7 @@
 #include <linux/skbuff.h>
 #include <linux/kthread.h>
 #include <linux/delay.h>
+#include <linux/sched/signal.h>
 #include "../platform/linux-dpaa2/kni/odpfsl_kni_common.h"
 #include "kni_fifo.h"
 #include "kni_dev.h"
-- 
2.7.4

